### PR TITLE
Kajam 11.3.0 plat 3963

### PR DESCRIPTION
**Pull request by hilak**

### CODE REVIEW

**Comment by hilak** [comment]
@assafb  @tan-tan-kanarek  Please review

**Comment by tan-tan-kanarek** [comment]
Please add dependency between comcast distribution provider plugin and cue-points and captions plugins.

**Comment by hilak** [comment]
Done.

File: plugins/content_distribution/providers/comcast_mrss/ComcastMrssDistributionPlugin.php [hunk 1]
```diff
@@ -22,8 +22,11 @@ public static function dependsOn()
 			self::CONTENT_DSTRIBUTION_VERSION_MINOR,
 			self::CONTENT_DSTRIBUTION_VERSION_BUILD);
 			
-		$dependency = new KalturaDependency(ContentDistributionPlugin::getPluginName(), $contentDistributionVersion);
-		return array($dependency);
+		$dependencyDistribution = new KalturaDependency(ContentDistributionPlugin::getPluginName(), $contentDistributionVersion);
+		$dependencyCaptions = new KalturaDependency(CaptionPlugin::PLUGIN_NAME);
```
Comment (tan-tan-kanarek) [inline plugins/content_distribution/providers/comcast_mrss/ComcastMrssDistributionPlugin.php]: Please create constants in current class, you can't use constants of other plugins before you know they're existing and enabled.

**Pull request merged**